app/profile/[username]/page.tsx

```diff
@@ -85,7 +92,7 @@ const Profile = async ({ params }: { params: { username: string } }) => {
       <h4 className="font-black uppercase pb-2">Top Songs Statistics</h4>
       <AudioFeatures audioFeatures={averageAudioFeatures} />
       <div className="divider" />
-      <div className="grid md:grid-cols-2 gap-4 pb-5">
+      <div className="grid md:grid-cols-3 gap-4 pb-5">
```

Comment:
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/7117993/229304611-646333dd-9e88-48a5-bd98-d1f3786c1c33.png">
Too compact, just do it at the bottom. You can also get rid of the `!isCurrentUser` that hides the recommended tracks, that way it's balanced